src/raftstore/store/store.rs

```diff
@@ -202,7 +202,8 @@ pub struct Store<T: Transport, C: PdClient + 'static> {
     pending_regions: Vec<metapb::Region>,
     split_check_worker: Worker<SplitCheckTask>,
     region_worker: Worker<RegionTask>,
-    compact_worker: Worker<CompactTask>,
+    raft_compact_worker: Worker<CompactTask>,
+    manual_compact_worker: Worker<CompactTask>,
```

Comment:
For example, when saying compact raft log, it just delete all the old raft log entries; when saying compact rocksdb, it means triggering rocksdb's compaction job.